src/app/components/Container.tsx

```diff
@@ -4,12 +4,12 @@ interface ContainerTypes {
 }
 
 const Container = ({ children, className }: ContainerTypes) => {
-    const classesName = `${className} flex-auto flex flex-col flex-initial 
-                                    bg-gray-400 px-5 py-3 h-5/6
-                                    rounded-lg bg-opacity-20 
-                                    backdrop-blur-lg
-                                    hover:shadow-lg hover:shadow-blue-950`;
+    const classesName = `${className} min-h-full`;
     return <div className={classesName}>{children}</div>;
```

Comment:
Good practice to use tailwindMerge (https://www.npmjs.com/package/tailwind-merge)